spec/menu_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish1) { double(:dish1, name: "Dishy1", price: 1.11) }
+  let(:dish2) { double(:dish2, name: "Dishy2", price: 2.22) }
+  let(:dish3) { double(:dish3, name: "Dishy3", price: 3.33) }
+  let(:dish_decimal) { double(:dish4, name: "Dishy4", price: 5.00) }
+
+  describe '#make_dish_available' do
+    it 'adds a new dish to the menu' do
+      subject.make_dish_available(dish1)
+      expect(subject.menu.values.include?(dish1)).to be true
+    end
+  end
+
+  describe '#view' do
+    it 'lists dishes available with price' do
+      subject.make_dish_available(dish1)
+      expected_output = "1: Dishy1 | Price: £1.11\n"
+      expect{ subject.view }.to output(expected_output).to_stdout
+    end
+
+    it 'lists multiple dishes with unique order numbers' do
+      subject.make_dish_available(dish1)
+      subject.make_dish_available(dish2)
+      subject.make_dish_available(dish3)
+      expected_output = "1: Dishy1 | Price: £1.11\n" +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.